doc/index.rst

```diff
@@ -137,7 +137,7 @@ Vectorizing a dataframe
 
    TableVectorizer
 
-Dirty Category encoders
+Dirty category encoders
 -----------------------
 
 .. autosummary::
```

Comment:
For some reason the `FeatureAugmenter` box is centered in the rendered docs, not sure why this is?